src/NServiceBus.AcceptanceTests/Serialization/When_sending_concrete_message_with_interface_props.cs

```diff
@@ -0,0 +1,69 @@
+﻿namespace NServiceBus.AcceptanceTests.Serialization
+{
+    using System;
+    using System.Threading.Tasks;
+    using AcceptanceTesting;
+    using EndpointTemplates;
+    using NUnit.Framework;
+
+    public class When_sending_concrete_message_with_interface_props : NServiceBusAcceptanceTest
```

Comment:
We already have those tests (added by @bording ). I'll just remove the acpt test since its redundant